tests/acceptance/features/apiSpaces/changingFilesShare.feature

```diff
@@ -0,0 +1,97 @@
+@api @skipOnOcV10
+Feature:
+
+  Background:
+    Given using spaces DAV path
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+  @issue-764
```

Comment:
This issue is related to old and new WebDAV only

Reply:
IMHO, there are no such DAV version specifications in the issue. Also, I don't think we should create a separate issue just for `spaces`.

Reply:
there is not any issue but there was a discussion before in [prevent the cross-space move](https://github.com/cs3org/reva/pull/3035)